docs/contrib/PlatformBringUp.md

```diff
@@ -0,0 +1,415 @@
+
+More customization can be done to increase the efficiency of global and detail route. Refer to the [OpenROAD documentation](https://openroad.readthedocs.io/_/downloads/en/latest/pdf/).
+
+
+## Metal Tracks Configuration
+
+OpenROAD requires a metal track configuration file for use in floorplanning. For each metal layer, the x and y offset as well as the x and y pitch are defined. To find the pitch and offset for both x and y, refer to the tech LEF. Following is a generalized metal tracks configuration file with five metal tracks defined.
+
+
+```
+# make_tracks.tcl
+###############################
+make_tracks metal1 -x_offset 0.24 -x_pitch 0.82 -y_offset 0.24 -y_pitch 0.82
+make_tracks metal2 -x_offset 0.28 -x_pitch 0.82 -y_offset 0.28 -y_pitch 0.82
+make_tracks metal3 -x_offset 0.28 -x_pitch 0.82 -y_offset 0.28 -y_pitch 0.82
+make_tracks metal4 -x_offset 0.28 -x_pitch 0.82 -y_offset 0.28 -y_pitch 0.82
+make_tracks metal5 -x_offset 0.28 -x_pitch 0.82 -y_offset 0.28 -y_pitch 0.82
+```
+
+
+Refer to the tech file to determine the x and y pitch and the offset.
+
+
+## PDN Configuration
+
+PDN is a utility that simplifies adding a power grid into the floorplan. With specifications given in the PDN configuration file, like which layer to use, stripe width and spacing, the utility can generate the metal straps used for the power grid. To create and configure a power grid, refer to [OpenROAD documentation](https://github.com/The-OpenROAD-Project/OpenROAD/blob/f864efbae4ffd4cc533f0e22c57cc7ab89618c44/src/pdn/doc/PDN.md) on pdngen.
+
+
+## Tapcell Configuration
+
+The tapcell configuration file is used to insert tapcells and endcaps into the design. Refer to the [Tapcell][https://github.com/The-OpenROAD-Project/OpenROAD/blob/f864efbae4ffd4cc533f0e22c57cc7ab89618c44/src/tap/README.md] documentation on how to construct this file.
+
+
+
+## setRC Configuration
+
+
+setRC allows the user to define resistances and capacitances for layers and vias using the `set_layer_rc` command. There is also a command that allows you to set the resistance and capacitance of routing wires using the `set_wire_rc`. Following is a generic example of a setRC configuration file which sets the resistance and capacitance of five metal layers, four vias, one signal wire, and one clock wire.
+
+
+```
+# setRC.tcl
+#######################
+set_layer_rc -layer M1 -capacitance 1.449e-04 -resistance 8.929e-04
+set_layer_rc -layer M2 -capacitance 1.331e-04 -resistance 8.929e-04
+set_layer_rc -layer M3 -capacitance 1.464e-04 -resistance 1.567e-04
+set_layer_rc -layer M4 -capacitance 1.297e-04 -resistance 1.567e-04
+set_layer_rc -layer M5 -capacitance 1.501e-04 -resistance 1.781e-05
+
+set_layer_rc -via V1 -resistance 9.249146E-3
+set_layer_rc -via V2 -resistance 4.5E-3
+set_layer_rc -via V3 -resistance 3.368786E-3
+set_layer_rc -via V4 -resistance 0.376635E-3
```

Comment:
What are the units for capacitance and resistance?

Reply:
cap and res are in Liberty units. The doc should make note of that.